pyproject.toml

```diff
@@ -76,3 +76,28 @@ include = [
   "/README.md",
   "/LICENSE.txt",
 ]
+
+[tool.pytest.ini_options]
+addopts = ["--strict-markers", "--strict-config"]
```

Comment:
```suggestion
addopts = ["--strict-markers", "--strict-config"]
python_classes = ["Test", "*TestCase"]
```

Reply:
Can we use noxfile.py for this? Then have a pull request that puts everything into addopts maybe?